listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -454,6 +454,111 @@ def fetch_recent_listens_for_users(self, users, min_ts: datetime = None, max_ts:
                 ))
         return listens
 
+
+    def fetch_recent_listens_for_users_new(self, users, min_ts: datetime, max_ts: datetime, limit=25):
```

Comment:
Instead of the new suffix, use some more informative name.

Reply:
Since this function is eventually going to replace the old `fetch_recent_listens_for_users` function, i thought a temporary new suffix should do. I'm not sure what to name it any suggestions?

Reply:
Changed function name and injected "all" in naming schema since the original function returned limited listens.